sw/device/tests/consecutive_irqs/consecutive_irqs_test.c

```diff
@@ -44,11 +44,11 @@ static const buffer_sink_t kPolledUartSink = {
     .sink = &polled_uart_sink_func,
 };
 
-static void debug_msg_and_abort(const char *msg) {
-  base_printf(msg);
-  base_printf("FAIL!\r\n");
-  abort();
-}
+#define LOG_FATAL(...) do { \
+  LOG_ERROR(__VA_ARGS__); \
+  base_printf("FAIL!\r\n"); \
+  abort(); \
+} while(false)
```

Comment:
I'd treat LOG_FATAL the same way as LOG_ERROR - its just a message with higher severity. 
Instead, we need a separate set of API to signal the end of test (I will put together something soon).

Reply:
Yeah, this is a leftover from a test Silvestrs wrote, and I've very much like a better API. I think we should keep this placeholder for now and switch once we have something concrete.